### PR TITLE
exp1 implementation

### PR DESCRIPTION
**Pull request by architag**

### CODE REVIEW

File: exp1.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
```
Comment (gagan93jtg) [inline exp1.html:1]: pls replicate design properly, compare using pixel perfect plugin

**Review by gagan93jtg — APPROVED** [review]

File: exp1_style.css [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+	margin: 10px;
+}
+#uname_image {
+	height: 20px;
+	vertical-align: middle;
+}
+#pass_image {
+	height: 25px;
+	vertical-align: middle;
+}
```
Comment (gagan93jtg) [inline exp1_style.css:54]: 1. Read about red symbol and how to remove.
2. What is the use of .DS_Store file?

File: exp1_style.css [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+#username {
+	border-color: rgb(237, 237, 237);
+	border-style: none none solid none;
+	padding: 5px;
+	margin-bottom: 28px;
+	width: 55%;
+	font-size: 15px;
+}
+#password {
+	border-color: rgb(237, 237, 237);
```
Comment (gagan93jtg) [inline exp1_style.css:34]: use 2 space indentation, not tab

**Review by gagan93jtg — CHANGES_REQUESTED** [review]

**Comment by architag** [comment]
> @architag: Why did we merge this PR??

Sir, I changed the indentation and newline at the end of file. Also I thought it said changes were approved.

**Pull request merged**